CTP/doc/sql/sql_guide.md

```diff
@@ -0,0 +1,450 @@
+How to view qahome results:  
+Go to QA homepage and click the CI build, wait for the page loading, then click the 'Function',look at the case of Fail  
+![qa_result](./qa_result.png)
+
+* Code coverage test  
+	* VERIFY CODE COVERAGE TESTING RESULT  
+	Go to QA homepage and find the ‘code coverage’ node in the left area, click the link of latest result.  
+	![coverage](./coverage.png)  
+	Click the Category(sql,medium,sql_by_cci) link.   
+	![category](./category.png)  
+	There is a coverage rate of lines. Its sql and sql_by_cci coverage rate of lines is usually in 58%~60%, the medium coverage rate of lines is usually in 30%~31%.  
+	![coverage_result](./coverage_result.png)
+
+	* SEND CODE COVERAGE TESTING MESSAGE  
+	sh  sender_code_coverage_testing_message.sh [QUEUE_NAME]  [COVERAGE_BUILD] [Category] default
+	eg:
+      ```
+      cd manual
+      sh  sender_code_coverage_testing_message.sh QUEUE_CUBRID_QA_SQL_LINUX_GIT_test http://192.168.1.91:8080/REPO_ROOT/store_01/10.2.0.8270-c897055/drop/CUBRID-10.2.0.8270-c897055-gcov-Linux.x86_64.tar.gz http://192.168.1.91:8080/REPO_ROOT/store_01/10.2.0.8270-c897055/drop/cubrid-10.2.0.8270-c897055-gcov-src-Linux.x86_64.tar.gz  sql
+      ```
+* Report issues  
+	* SQL ISSUE  
+	You can refer to http://jira.cubrid.org/browse/CBRD-22721   
+    ![issues1](./issues1.png)  
+    ![issues2](./issues2.png)  
+	It is also necessary to add this information in the comment to facilitate the developers to find the information they need. Note that choose Restricted to Developers since they have they contain sensitive information such as IP, port, password, etc..  
+* how to verify issue?  
+	```
+	Test Build: 10.2.0.8239-1f051a0 debug
+	Test OS: Linux 64bit
+	Test Result: Pass.
+	Add test case for this issue: 
+
+	Close this issue.
+	```
+# 5. test case specification  
+* When add a test case for a bug fix, add a case to path  
+	cubrid-testcases/sql/_13_issues/_{yy}_{1|2}h/cases with naming rules:  
+	    cbrd_xxxxx.sql  
+	    cbrd_xxxxx_1.sql  
+	    cbrd_xxxxx_2.sql  
+	    cbrd_xxxxx_xasl.sql  
+
+
+* When add a test case for a feature test, add a case to path  
+	cubrid-testcases/sql/_{no}_{release_code}/cbrd_xxxxx_{feature}/cases with naming rules:   
+	    {any_structured_name}.sql
+
+* How to make a SQL test case  
+	A SQL test cases follows following basic structure:  
+	any_testcase_folder/cases/case_file.sql  
+	any_testcase_folder/answers/case_file.answer  
+	any_testcase_folder/answers/case_file.answer_WIN (If the answers are different in Windows)  
+	any_testcase_folder/answers/case_file.answer_cci (If the answers are different in CCI)  
+  
+	A SQL file goes to 'cases' folder, an answer file goes to 'answers' folder, and they should be in the same folder.The test case and answer should have the same name.
+ 
+* Notes for writing case:  
+  * Drop table before and after test  
```

Comment:
Move all following to upper level.  Give each concrete syntax. (Remove "Notes for writing case" level.)

"Select ... order by". Please tell to user that why we need this.

There should be more rules. Please add all of them. For example, how to add "query plan", etc.